lib/Futures/include/Futures/coro-helper.h

```diff
@@ -34,6 +34,7 @@ namespace std_coro = std;
 #include "Basics/Result.h"
 #include "Promise.h"
 #include "Try.h"
+#include "Utils/ExecContext.h"
 
```

Comment:
This whole class needs an explanation in the source code, such that future developers (including our own future selves) can understand what it is for.

Reply:
Done.